### ISSUE TITLE
我需要帮助，我在训练过程中遇到了非常大的问题

### ISSUE THREAD

**Issue by LuckyFrances** [ptran1203/pytorch-animeGAN#31]
生成的图片完全不具有动漫风格，请问该怎么解决呢

**Comment by capp-adocia** [comment]
@LuckyFrances Generally speaking, after training for about 20 epochs according to the given parameters and resources, you can find that the generated images have an anime style. I am not sure what you mean. Please describe what happened in more detail. Also, when asking questions in the community, it is best to use international languages : )

**Comment by LuckyFrances** [comment]
Thank you very much for your reply! The main problems I encountered are as follows: the content loss and color loss of the generator have been increasing, and the final generated images are roughly as follows: P1 is trained for 20 rounds, P2 is trained for 70 rounds, and they clearly do not have anime style. During training, I also adjusted the weights of different training functions according to the training style, but this problem still occurred. May I ask where I went wrong?

<img width="984" alt="Image" src="https://github.com/user-attachments/assets/e17694cf-b013-440a-ad7e-19138c4e58a2" />

<img width="483" alt="Image" src="https://github.com/user-attachments/assets/12dd5a25-71ae-497d-ab0b-b033c1a376c5" />

**Comment by Porizm** [comment]
I have the same problem too, can you please give us the proper settings for training?

**Comment by capp-adocia** [comment]
To be honest, I'm sorry that I haven't touched this for a while.

But I remember that the final trained model had a little anime style.

Unfortunately, no matter how I adjusted the parameters, I couldn't achieve the anime effect like in the document. I guess it might have something to do with the training dataset.

I once communicated with the author of AnimeGAN.. This is his email address `asher_chan@foxmail.com`, maybe he can provide some suggestions.

**Comment by LuckyFrances** [comment]
Hello, I am also reproducing the model of animeganv2. If possible, could you add me on WeChat or QQ? We can communicate together. This is my WeChat ID: fs1970298461




1970298461
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ptran1203/pytorch-animeGAN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年3月31日(星期一) 晚上8:26
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ptran1203/pytorch-animeGAN] 我需要帮助，我在训练过程中遇到了非常大的问题 (Issue #31)



   
I have the same problem too, can you please give us the proper settings for training?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***&gt;
  Porizm left a comment (ptran1203/pytorch-animeGAN#31)
 
I have the same problem too, can you please give us the proper settings for training?

**Comment by LuckyFrances** [comment]
Thank you for your reply! Actually, I don't need to reproduce it like the document at the moment. The main issue is that the losses are constantly increasing and the final image style that appears is quite strange (in fact, I only need it to have an anime style, but it seems to be completely absent now) So I suspect there may be something wrong. Thank you, I will try to find out where I went wrong

**Comment by ptran1203** [comment]
I wrote a comment may relate to this [here](https://github.com/ptran1203/pytorch-animeGAN/issues/30#issuecomment-2769733501)
